watertap/examples/flowsheets/case_studies/groundwater_treatment/tests/test_groundwater_treatment_sweep.py

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import os
+import pytest
+from watertap.examples.flowsheets.case_studies.groundwater_treatment import (
+    groundwater_treatment_sweep,
+)
+
+sweep_list = []
+for case_num in [1, 2, 3]:
+    sweep_list.append(case_num)
+
+
+@pytest.mark.parametrize("case_num", sweep_list)
+@pytest.mark.integration
+def test_sweep(case_num, tmp_path):
+    cwd = os.getcwd()
+    os.chdir(tmp_path)
+    nx = 11
```

Comment:
```suggestion
    nx = 2
```
Don't see any reason to run more, given we don't check the outputs.